docs/content/token/staking/transactions.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Staking Transactions Reference
+sidebar_title: Staking Transactions Reference
```

Comment:
```suggestion
```

`sidebar_title` is optional -- it overrides the value of `title`, so isn't needed in this case